src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -973,7 +974,13 @@ bool emberAfOperationalCredentialsClusterCSRRequestCallback(app::CommandHandler
         VerifyOrExit(csr.Alloc(csrLength), err = CHIP_ERROR_NO_MEMORY);
 
         err = gFabricBeingCommissioned.GetOperationalKey()->NewCertificateSigningRequest(csr.Get(), csrLength);
-        emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: NewCertificateSigningRequest returned %" CHIP_ERROR_FORMAT, err.Format());
+        if (err != CHIP_NO_ERROR)
+        {
+            ChipLogError(Zcl, "OpCreds: NewCertificateSigningRequest returned %" CHIP_ERROR_FORMAT, err.Format());
+            SuccessOrExit(err);
+        }
+        ChipLogProgress(Zcl, "OpCreds: NewCertificateSigningRequest succeeded");
+
         SuccessOrExit(err);
```

Comment:
```suggestion
```
since now we have it inside the `if (err != CHIP_NO_ERROR)` block, right?

Reply:
Done